sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateBoxDao.java

```diff
@@ -41,10 +41,11 @@
 @Repository
 public class HibernateBoxDao implements BoxStore, HibernatePaginatedDataSource<Box> {
 
-  public static final String[] SEARCH_PROPERTIES = new String[] { "name", "alias", "identificationBarcode", "locationBarcode" };
+  private static final String FIELD_ALIAS = "alias";
 
-  private final static List<String> STANDARD_ALIASES = Arrays.asList("lastModifier",
-      "creator", "size", "use");
+  public static final String[] SEARCH_PROPERTIES = new String[] { "name", FIELD_ALIAS, "identificationBarcode", "locationBarcode" };
+
+  private final static List<String> STANDARD_ALIASES = Arrays.asList("lastModifier", "creator", "size", "use");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)